### DISCUSSION TITLE
Mapsui with a GPU backend

### DISCUSSION THREAD

**Discussion by charlenni** [Mapsui/Mapsui#2347] [Q&A]
I'm searching around how to use Mapsui on a Windows desktop app with a GPU backend, but didn't find something. I can create an app with WinForms, but that seems not right in these times. For WPF SkiaSharp has only a CPU backend. The same to WinUI. UWP has a GPU backend, but is deprected.

So I think, that WinForms is the only Windows platform, that support GPU nowadays. Is this correct?

**Comment by pauldendulk** [comment]
I don't know what the current situation is. You would expect SkiaSharp to support GPU on Windows since it is maintained by Microsoft. Although the limiting factor could be skia itself. Perhaps you can ask this in the SkiaSharp forum.

**Comment by der1Mac** [comment]
In Scottplot they use a SkiaSharp.Views.WPF.SKGLElement they copied from the SkiaSharp repository. See here: https://github.com/ScottPlot/ScottPlot/pull/3008. This element seems to be not yet in the official SkiaSharp release. I have no knowledge on details so I can't help any further.

**Reply by pauldendulk** [reply]
Hey thanks for that info!
